Makefile

```diff
@@ -8,9 +8,9 @@ build-kubernetes:
 	docker build -f _infra/docker/Dockerfile .
 
 lint:
-# remove -i 70624 once flask-cors is updated beyond v4.0.1 to solve vulnerability
+# remove -i 70624 and -i 72731 once flask-cors is updated beyond v4.0.1 to solve vulnerability
```

Comment:
Flask-cors has been updated to 5.0.0 in the pipfile.lock?

Reply:
Yes, I forgot to update the note. It's corrected now.